update_compilers/install_python_compilers.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+. ${SCRIPT_DIR}/common.inc
+
+get_python() {
+    local VER=$1
+    local DIR=Python-$VER
+
+    if [[ ! -d ${DIR} ]]; then
+        mkdir $DIR
+        pushd $DIR
+
+        fetch "https://www.python.org/ftp/python/${VER}/Python-${VER}.tgz" | tar xfz -
+        cd $DIR
+
+        ./configure --prefix=${OPT}/python --exec-prefix=${OPT}/python --without-ensurepip
```

Comment:
I'm not entirely sure how I feel about all python versions sharing the same directory. Nothing else currently does this, and although we don't currently ever _remove_ any compilers if we did decide to in the future this would make things difficult.

Reply:
Every version keeps its files independently tagged, which is why I did it this way, but that makes sense. I can swap it over pretty easily, it's just changing the prefix to end with $VER